www/shims/jquery.html.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+/**
+ * @param {FetcherIndex} index to use for URL resolution
+ * @param {Object} context e.g. jQuery, $, etc
+ * @param {String} method name e.g. 'ajax'
+ * @returns {Function} the shim
+ */
+
+//var cheerio = require('cheerio');
+
+module.exports = function (index, context, method) {
+  var oldMethod = context[method];
+
+  var fixSrc = function (whole, value) {
+    console.log('input whole is: '+ whole + ', value is: ' + value);
+    var fixValue;
+    if (value && typeof value === 'string') {
+      fixValue = index.resolveLocalUrl(value);
+      whole = whole.replace(value, fixValue);
+    }
+    console.log('fixed whole is: '+ whole + ', value is: ' + fixValue);
```

Comment:
Yes, I tested with or without log, I was kind of feeling that printing those console logs could help a little for our likely happen timing issue. But I just tested again, looks like no big difference. Already removed.

> On 10 Aug 2015, at 11:04 am, Ron Waldon <notifications@github.com> wrote:
> 
> In www/shims/jquery.html.js <https://github.com/jokeyrhyme/appcache-fetcher.js/pull/5#discussion_r36596550>:
> 
> > + * @returns {Function} the shim
> > + */
> > +
> > +//var cheerio = require('cheerio');
> > +
> > +module.exports = function (index, context, method) {
> > +  var oldMethod = context[method];
> > +
> > +  var fixSrc = function (whole, value) {
> > +    console.log('input whole is: '+ whole + ', value is: ' + value);
> > +    var fixValue;
> > +    if (value && typeof value === 'string') {
> > +      fixValue = index.resolveLocalUrl(value);
> > +      whole = whole.replace(value, fixValue);
> > +    }
> > +    console.log('fixed whole is: '+ whole + ', value is: ' + fixValue);
> Another one?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/jokeyrhyme/appcache-fetcher.js/pull/5/files#r36596550>.
>